### DISCUSSION TITLE
v2/md-enhance/zh/guide/tasklist

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#395] [Announcements]
# v2/md-enhance/zh/guide/tasklist

让你的 VuePress 站点中的 Markdown 文件支持任务列表。 配置 tab TS tab JS 语法 使用 - [ ] 一些文字 渲染一个未勾选的任务项; 使用 - [x] 一些文字 渲染一个勾选了的任务项 (我们也支持大写的 X); 例子 [ ] Plan A; [x] Plan B; 高级 除了在插件选项中设置 tasklist: tr...

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/tasklist.html

**Comment by jxcsoft** [comment]
希望这个任务列表能实现勾选之后显示不同的内容
- [ ] 显示"A"
- [x] 显示"B"

**Reply by Mister-Hope** [reply]
你可以自己做一个组件，我们对于内容切换的情形设计了“选项卡”功能。

小众的定制化需求都请自己实现。如果你觉得有一类情形没能被满足欢迎到仓库开issue
